tests/network/vmi_multus.go

```diff
@@ -170,8 +170,14 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 		nodes = util.GetAllSchedulableNodes(virtClient)
 		Expect(len(nodes.Items) > 0).To(BeTrue())
 
-		const vlanId = 100
-		Expect(createBridgeNetworkAttachmentDefinition(util.NamespaceTestDefault, linuxBridgeVlan100Network, vlanId, "")).To(Succeed())
+		const (
+			vlanId                = 100
+			bridgeCNIType         = "bridge"
+			bridgeName            = "br10"
+			macSpoofCheckDisabled = false
```

Comment:
I think these need to appear in a place where it can be used by both usages.
And the name needs to be fixed to the value, I mean:
```
vlanID100 = 100
bridge10 = "br10"
```

Reply:
Done